v_4_4_0/files/k8s-resource/calico-all.yaml

```diff
@@ -420,6 +423,7 @@ spec:
         - name: etcd-certs
           hostPath:
             path: /etc/kubernetes/ssl/etcd
+            defaultMode: 0400
```

Comment:
Upstream uses a secret for this, trying to change the mode with our mounted file as well :man_shrugging: